### DISCUSSION TITLE
Coupled Newton-Krylov iteration

### DISCUSSION THREAD

**Discussion by tbellosta** [su2code/SU2#1992] [Ideas]
I am interested in coupling the linear system solve for the different solvers in the RANS iteration as it may be able to give a sensible speed-up of the time-to-solution (for the Euler equations I am getting close to 10x speed-up).
Is anyone else interested / already working on it /  want to collab?

**Comment by pcarruscag** [comment]
At this point: https://github.com/su2code/SU2/pull/1183/commits/8732e5d27fe7a2ca07b88dab2b7a769eca0c55bf I had a coupled implementation, then I took it out and kept it only for fluid because the linear system was too difficult to solve.
